.circleci/utils/test_sort_yaml.py

```diff
@@ -11,4 +11,4 @@
 import sys
 import yaml
 
-sys.stdout.write(yaml.dump(yaml.load(sys.stdin, Loader=yaml.FullLoader), sort_keys=True))
+sys.stdout.write(yaml.dump(yaml.safe_load(sys.stdin, Loader=yaml.FullLoader), sort_keys=True))
```

Comment:
Thank you for the fix, but again, this file is not part of `torchvision` package